test/e2e/resources/usergroup_additional_user.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: elasticache.services.k8s.aws/v1alpha1
```

Comment:
If you are able to patch the resource as indicated in the comment below, we may be able to remove this file